ProcessLib/ThermoMechanicalPhaseField/ThermoMechanicalPhaseFieldProcess-impl.h

```diff
@@ -0,0 +1,349 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <cassert>
+
+#include "NumLib/DOF/ComputeSparsityPattern.h"
+#include "ProcessLib/SmallDeformation/CreateLocalAssemblers.h"
+#include "ProcessLib/Process.h"
+
+#include "ThermoMechanicalPhaseFieldFEM.h"
+#include "ThermoMechanicalPhaseFieldProcessData.h"
+#include "ThermoMechanicalPhaseFieldProcess.h"
```

Comment:
Move it to the line before  `#include <cassert>`. In other words, it should be the first include.

Reply:
done.